src/dumbo/src/tcp/connection.rs

```diff
@@ -1548,18 +1549,23 @@ pub(crate) mod tests {
         let remote_isn = t.remote_isn;
         let mss = u32::from(t.mss);
 
+        let (payload_buf, mut response_seq) = payload_src.unwrap();
```

Comment:
Just wondering,  do these changes and the ones in the following block do anything compared to the previous version of the code?

Reply:
This is an example of accounting which should be done by a higher level component of the stack, like our tcp `Endpoint`. It is not necessary, and I do not have a strong preference to keep it.